sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_models.py

```diff
@@ -382,10 +253,9 @@ class FileSystemSasPermissions(object):
     :keyword bool create:
         Write a new file, snapshot a file, or copy a file to a new file.
     :keyword bool move:
-        Move any file in the directory to a new location.
-        Note the move operation can optionally be restricted to the child file or directory owner or
-        the parent directory owner if the saoid parameter is included in the token and the sticky bit is set
-        on the parent directory.
+        Move any file in the directory to a new location. Note the move operation can optionally be restricted to the
+        child file or directory owner or the parent directory owner if the said parameter is included in the token
+        and the sticky bit is set on the parent directory.
```

Comment:
Can you just verify this isn't messed up? I can't tell if this is a GitHub rendering issue but it looks like there's a weird line break here or something
![image](https://github.com/user-attachments/assets/b61fffb3-2386-4767-b661-5249faeb2cd1)

Reply:
Looks fine for me :). Got to get a wide monitor to see it properly

Reply:
Confirming that it looks good in the IDE, here, as well as the docs rendering!

Reply:
![Screenshot 2024-10-21 at 10 45 12 AM](https://github.com/user-attachments/assets/e4c1e219-2e6f-4ad1-aab5-2e3252aaf892)